ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/user-labels-tags/cucushift-installer-rehearse-gcp-ipi-user-labels-tags-workflow.yaml

```diff
@@ -0,0 +1,25 @@
+workflow:
+  as: cucushift-installer-rehearse-gcp-ipi-user-labels-tags
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision
+    post:
+    - chain: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-deprovision
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
```

Comment:
suggest to move the `FEATURE_SET` into job config level, so that we can easily switch the job from tp to non-tp